src/reanimated2/Bezier.ts

```diff
@@ -52,16 +52,16 @@ export function Bezier(
     aX: number,
     aA: number,
     aB: number,
-    mX1: number,
-    mX2: number
+    innerMX1: number,
```

Comment:
I think this should be a suffix, not a prefix. While I understand that semantically it looks a lot better and is a better resemblance of a natural language, you are polluting the `inner` namespace, e.g. autocompletion will suggest a lot of things. @piaskowyk, @m-bert  what do you think?

Reply:
I don't mind it being prefix. It looks more natural this way.

Reply:
I agree, `inner` isn't the best prefix in this case